test/integration.test.ts

```diff
@@ -567,6 +567,55 @@ describe("authentication", () => {
     }));
 });
 
+describe("queries", () => {
+  test("runQuery", () =>
+    Effect.gen(function* () {
+      const c = yield* TestConvexService;
+
+      yield* Effect.all([
+        c.run(({ db }) => db.insert("notes", { text: "Hello, world!" })),
+        c.run(({ db }) => db.insert("notes", { text: "Hello, world!" }))
+      ])
+
+      const notes = yield* c.query(api.functions.runQueryOnly);
+
+      expect(notes).toMatchObject([
+        {
+          _id: '10000;notes',
+          text: 'Hello, world!'
+        },
+        {
+          _id: '10001;notes',
+          text: 'Hello, world!'
+        }
+      ]);
+    }),
+  );
+});
+
+describe("mutations", () => {
+  test("runQueryAndMutation", () =>
```

Comment:
I'd name this more closely after the thing we're actually testing, which here is `runQuery` and `runMutation`.